Jenkinsfile

```diff
@@ -55,6 +56,11 @@ pipeline {
                         }
                     }
                 }
+		stage ("Build Docs") {
+		    steps {
+		        sh 'TOXDIR=/tmp/$(basename ${WORKSPACE})/tox-docs make docs'
+		    }
+		}
```

Comment:
I have a feeling, but can't find evidence to back it up, that even doing this doesn't prevent parallel tox sadness. Fortunately it doesn't matter in recent versions of tox, because builds are all parallel safe anyway. Unfortunately the slaves this is set to run on have tox 3.0.0 (latest is 3.7.0, parallel safety came in [3.5.0](https://tox.readthedocs.io/en/latest/changelog.html#v3-5-0-2018-10-08)).
Should we go with this for now, and think about how much pain upgrading tox on the slaves will cause?